src/main/webapp/style.css

```diff
@@ -148,3 +148,33 @@ ul {
   padding: 1rem 0;
   text-align: center;
 }
+
+.hiddenButton {
+  border: solid #333;
```

Comment:
Specify border size using <code>rem units</code>. Rem units are a ratio of the font size declared in the <code>:root</code> (16px), e.g. 1rem = 16px.